crates/turborepo-lib/src/shim.rs

```diff
@@ -317,8 +317,14 @@ impl LocalTurboState {
     // - `pnpm install`
     // - `npm install --install-strategy=linked`
     fn generate_linked_path(root_path: &AbsoluteSystemPath) -> Option<AbsoluteSystemPathBuf> {
-        let canonical_path =
-            fs_canonicalize(root_path.join_components(&["node_modules", "turbo", ".."])).ok()?;
```

Comment:
Detailed tests caught that this was not materially the same behavior as the original.